### PR TITLE
Make liquidMaterialToDensityMap public

### PR DESCRIPTION
**Pull request by Gotve1**
## What this PR does:
- [ ] Bug fix 
- [ ] Feature
- [+] Code refactor / improvement
- [ ] API addition / improvement
- [ ] Compat with another mod

**Explain what this PR is doing:**
makes liquidMaterialToDensityMap public until someone fix MassDatapackResolver

---

## Modloaders this PR affects:
- [+] Forge
- [+] Fabric

## Where this PR was tested:
- [+] In-dev singleplayer
- [ ] In-dev dedicated server
- [ ] Out of dev singleplayer
- [ ] GameTest framework

## Breaking Changes
- [ ] Yes (describe)
- [+] No

---

## Additional Notes
Anything else reviewers might need to know:
Performance concerns, edge cases, etc

---

## Declaration
By submitting this PR, I affirm:  
- Code/data compiles and loads  
- Tests _(if any)_ pass  
- This PR will not brick worlds

### CODE REVIEW

**Comment by blockninja124** [comment]
Wouldn't it be better to just... identify and fix the issue with the datapack resolver?

File: common/src/main/kotlin/org/valkyrienskies/mod/common/config/MassDatapackResolver.kt [hunk 1]
```diff
@@ -277,6 +277,15 @@ object MassDatapackResolver : BlockStateInfoProvider {
      *
      * It is **not recommended** to call this yourself!
      */
+
+    val liquidMaterialToDensityMap: HashMap<Fluid, Pair<Double, Double>> = hashMapOf(
```
Comment (blockninja124) [inline common/src/main/kotlin/org/valkyrienskies/mod/common/config/MassDatapackResolver.kt:281]: I don't think this is even a mutable map, so there isn't any point to exposing it? Addons will just be able to read it, not write to it.

Comment (zyxkad) [reply common/src/main/kotlin/org/valkyrienskies/mod/common/config/MassDatapackResolver.kt:281]: HashMap is mutable

Comment (blockninja124) [reply common/src/main/kotlin/org/valkyrienskies/mod/common/config/MassDatapackResolver.kt:281]: In kotlin? You usually have to use `mutableMapOf()` instead of `mapOf()`, but I don't know if that applies to hash maps.
